Fix kcoin crash issue

When restarting the genesis validator on the testnet with around 500k blocks, this error keeps coming up:

```
INFO [06-15|09:50:31] Imported new block receipts              hostname=genesis-validator 
INFO [06-15|09:50:31] Imported new block receipts              hostname=genesis-validator 
INFO [06-15|09:50:31] Imported new block receipts              hostname=genesis-validator 
INFO [06-15|09:50:31] Imported new block receipts              hostname=genesis-validator 
INFO [06-15|09:50:31] Imported new block receipts              hostname=genesis-validator 
INFO [06-15|09:50:31] Imported new block receipts              hostname=genesis-validator 
INFO [06-15|09:50:31] Imported new block receipts              hostname=genesis-validator 
INFO [06-15|09:50:31] Imported new block receipts              hostname=genesis-validator 
panic: runtime error: slice bounds out of range

goroutine 104 [running]:
github.com/kowala-tech/kcoin/stats.(*Service).reportHistory(0xc4201f7a40, 0xc4202c1560, 0x0, 0x0, 0x0, 0x0, 0x0)
    /kcoin/build/_workspace/src/github.com/kowala-tech/kcoin/stats/stats.go:562 +0x8e5
github.com/kowala-tech/kcoin/stats.(*Service).loop(0xc4201f7a40)
    /kcoin/build/_workspace/src/github.com/kowala-tech/kcoin/stats/stats.go:232 +0xeaf
created by github.com/kowala-tech/kcoin/stats.(*Service).Start
    /kcoin/build/_workspace/src/github.com/kowala-tech/kcoin/stats/stats.go:103 +0x5c

```

This issue it hard to replicate.

--- a/client/stats/stats.go
+++ b/client/stats/stats.go
@@ -560,6 +560,7 @@ func (s *Service) reportHistory(conn *websocket.Conn, list []uint64) error {
 		}
 		// Ran out of blocks, cut the report short and send
 		history = history[len(history)-i:]
+		break
 	}
 	// Assemble the history report and send it to the server
 	if len(history) > 0 {

